pkgs/development/python-modules/aiohttp-fast-url-dispatcher/default.nix

```diff
@@ -30,6 +30,8 @@ buildPythonPackage rec {
 
   nativeBuildInputs = [ poetry-core ];
 
+  pythonRelaxDeps = [ "aiohttp" ];
```

Comment:
Note that Home Assistant removed the dependency in https://github.com/home-assistant/core/pull/122409 so we might as well drop this package.

Reply:
Sure, why not.

Reply:
Let's do so once the next Home Assistant update is merged.